docs/concepts/did.md

```diff
@@ -2,3 +2,103 @@
+There is a one-to-one relationship between DIDs and Web3 names.
+This means that a KILT DID can be linked to at most one Web3 name, and a Web3 name can be claimed only by one DID.
+
+Each Web3 name is globally unique within the KILT space, and is composed of a sequence of a minimum of 3 to a maximum of 32 characters taken from a specific character set to enhance human redability and reduce the chances of "two Web3 names looking the same, despite being different".
+The character set include only:
+- *lowercase letters*, from `a` to `z`
+- *digits* from `0` to `9`
+- the symbols `-` and `_`
+
+A regex that would match all and only the allowed Web3 names would be the following:
+
+```
+^[a-z0-9\_\-]{3,32}$
+```
+
+In the global URI space, Web3 names are prefixed with the `w3n:` URI namespace. 
+For example, the full URI for the Web3 name `example-web3name` is `w3n:example-web3name`.
+
+### Linking accounts to a Web3 name
+
+Beyond linking a Web3 name, KILT allows DID owners to link multiple accounts to a single DID.
+These accounts are not specific to the KILT blockchain but can reference any chain within the Dotsama ecosystem.
+Each account <-> DID link requires the payment of a small deposit, which is returned whenever the link is removed.
+
+For DIDs that have also claimed a Web3 name, the linking feature opens up the way to a host of possibilities, e.g., showing the Web3 name of a collator's account on the [KILT stakeboard](https://stakeboard.kilt.io/).
+
+<ThemedImage
+  alt="DID lookup diagram"
+  sources={{
+    light: '/img/concepts/did/did-lookup-light.png',
+    dark: '/img/concepts/did/did-lookup-dark.png',
+  }}
+/>
+
+:::caution
+While multiple accounts can be linked to a DID, it is important to notice the difference between the two.
+
+KILT *accounts* are classical blockchain accounts, that can be used to hold and send KILT tokens, as well as signing and submitting transactions.
+On the other hand, KILT *DIDs* are a L2 construct which are derived from KILT accounts, but are completely separated from them.
+This means that **KILT DIDs cannot hold any KILT tokens**.
+DIDs are used to authorize (sign) some operations, but the resulting signature must then be submitted to the blockchain by a KILT account, which must pay for the transaction fees.
+
+Hence, even though a DID has the format `did:kilt:4rp4rcDHP71YrBNvDhcH5iRoM3YzVoQVnCZvQPwPom9bjo2e` with `4rp4rcDHP71YrBNvDhcH5iRoM3YzVoQVnCZvQPwPom9bjo2e` being a valid KILT account, there is no (immediate) relationship between the DID and this account and hence this account should not be considered in isolation as a KILT account, but only with its `did:kilt` prefix, which makes it a valid DID.
+Hence, if instructed to "*send some funds to the DID by using the account after the `did:kilt` prefix*", please ignore the advice, as sending funds to a DID can result in those funds being lost without the required technical expertise.
```

Comment:
You used hence twice in a row. 
```suggestion
Therefor, if instructed to "*send some funds to the DID by using the account after the `did:kilt` prefix*", please ignore the advice, as sending funds to a DID can result in those funds being lost without the required technical expertise.
```

Reply:
https://github.com/KILTprotocol/docs/pull/82/commits/f88c41076ae7f68ec87b4c5bef28de5cbb6051b0